etn-ccis/blui-react-design-patterns
# blui 3743 use github actions for CI

Fixes blui-3743 .


#### Changes proposed in this Pull Request:
- add blui-ci config to replace circleCI
- upgrade cypress
- remove firebase tools package


#### Screenshots / Screen Recording (if applicable)
-



#### To Test:
- you can commit a change to this PR on the readme and see blui-ci "CI-Run" action if you want
- or view what's happening here https://github.com/etn-ccis/blui-react-design-patterns/actions/runs/6149904797

#### Any specific feedback you are looking for?
- The codecov report config is slightly different than the old reports. So I will need to do a new compare once this gets merged
